tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+pytest_plugins = [
+    "tests.fixtures.fixture_preprocess_text",
```

Comment:
Does this mean _all_ our tests are importing this fixture?

Reply:
Oooo, good point, yes, I think so. Given that it is quite a large fixture, then not a very good idea. I'll look at confining its scope to my specific test.

Reply:
I'm not clear from the docs when this fixture gets loaded, so forgive me for asking but just to check: so now it only gets loaded for the test that we call it for?

Reply:
Yes, I think so. The pytest docs are quite bad, I'm not fully clear on it myself either. 😓

Reply:
Cool let's go with it then.